doc/geany.txt

```diff
@@ -4593,6 +4593,11 @@ calltips
 
     *Example:* ``calltips=0xc0c0c0;0xffffff;false;false``
 
+indicator_error
+    The color of the error indicator.
+
```

Comment:
If you want, but as most of these settings use the value differently I thought "the color" was clear enough.  Do you want me to add something like *"Only the first argument (foreground color) is used."*?